### PR TITLE
refine new config design

### PR DESCRIPTION
**Pull request by chai2010**
keep use same env name as before

### CODE REVIEW

File: pkg/config/api/api_config.go [hunk 1]
```diff
@@ -50,17 +53,25 @@ type RepoService struct {
 	Port int    `default:"9104"`
 }
 
-func LoadApiConfig() (*ApiConfig, error) {
-	p := new(OpenPitrix_Api_Config)
+func MustLoadConfig() *Config {
```
Comment (chilianyi) [inline pkg/config/api/api_config.go]: Can we put these common func into utils fold, so multiple module will share them.

Comment (chai2010) [reply pkg/config/api/api_config.go]: it is just has the same name, the real config for each service is different.

for example: 
- the api service do not need database
- the app service do not need api host and port, and the db user name is different from other service's db user

only the glog config is same for all service.

Comment (chilianyi) [reply pkg/config/api/api_config.go]: MustLoadConfig and LoadConfig 
I saw the two func are the same between these module .

Comment (chai2010) [reply pkg/config/api/api_config.go]: only the `loadConfig` and `Glog` are same.
The LoadConfig's return type is OpenPitrix_Config, it is different for each service.
You should check the `OpenPitrix_Config` for these module.

**Pull request merged**